### PR TITLE
My profile page

### LINKED ISSUE #55: E: 5. As an authorized user  want to have my Profile Page

### PR DESCRIPTION
**Pull request by RMuskovets**
# Checklist:

- [ ] Added specs for all new ruby code
- [x] Green rspec run
- [x] Green rubocop
- [x] PR review from teammate

# What?
Added a My Profile page.

## Why?
Because there was such task... And well, if users have information about them, there should be a way to view it.

## How?
Used the `show` action in `UsersController` and made a view for it (`app/views/users/show.html.slim`). For styling used Bootstrap and a bit of custom SCSS (`app/frontend/stylesheets/users.scss`).

## Testing?
Did you write tests? No.

## Screenshots (optional)
Screenshot 2021-06-13 at 21-36-54 Roman HostMePlease

## Anything Else?
Closes #55. (finally...)

### CODE REVIEW

File: app/frontend/stylesheets/fonts.scss [hunk 1]
```diff
@@ -1,2 +1,6 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;700;800&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Material+Icons');
+
+.montserrat {
+    font-family: 'Montserrat';
+}
```
Comment (KostyaKobel) [inline app/frontend/stylesheets/fonts.scss:6]: Add new empty line

File: app/views/users/show.html.slim [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          tbody
+            tr
+              td Name:
+              td = @user.first_name
+            tr
+              td Email:
+              td = @user.email
+            tr
+              td Phone:
+              td = "+380 68 123 56 78"
```
Comment (KostyaKobel) [inline app/views/users/show.html.slim:26]: Add new empty line

**Review by VanyaNovak — APPROVED** [review]
LGTM!

**Review by KostyaKobel — APPROVED** [review]
Good!

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
+  before_action :user_find, only: [:show]
 
   def index
     redirect_to root_path
   end
 
-  def show
-    user_find
-  end
+  def show; end
```
Comment (andrjuha01) [inline app/controllers/users_controller.rb:9]: there are no specs for this action

Comment (RMuskovets) [reply app/controllers/users_controller.rb:9]: wrote some

File: config/environments/test.rb [hunk 4]
```diff
@@ -42,7 +42,7 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
-
+
```
Comment (andrjuha01) [inline config/environments/test.rb:45]: remove unneeded spaces

File: spec/factories/users.rb [hunk 5]
```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :user do
+    sequence(:id) { |n| n }
```
Comment (andrjuha01) [inline spec/factories/users.rb:3]: please don't generate id's like that. 
It's database responsibility to keep id list consistent

**Review by andrjuha01 — APPROVED** [review]

**Pull request merged**